### ISSUE TITLE
website statistics incorrect wording

### ISSUE THREAD

**Issue by osusoy** [bluethrust/clanscripts#14]
[bug] didn't notice this while i was setting up (i was the only one using it) but one of the new members pointed it out; on the home page (with the clocks), hit counter says "You have viewed this site xxxx times.". However, the number displayed seems to actually be the total hit count, not a particular user's so wording along the lines of "This site has been viewed xxxx times" would be more appropriate.

**Comment by osusoy** [comment]
clarification. what is displayed:

```
Total Page Views: 2628
Total Unique Views: 1

You have viewed this site 2628 times.
```

Total page views is likely to be accurate, i was refreshing quiet a bit while looking into bugs. But, there has been 2 unique views which makes that part incorrect and the other member is reporting "You have viewed this site" part to be showing the same number as mine. it almost seems like the counters are treating everyone as one and i have an idea why it might be doing this:

The way I have my servers setup is that all requests go to a single server running NginX and its configuration routes the request to the correct server (so is basically a proxy server). It adds certain headers to avoid confusion and pretty much all web apps i have used deal with this just fine. But, the behavior above tells me that you are not checking for a X-Real-IP or X-Forwarded-For headers which would mean that Bluethrust would consider everyone's IP to be that of my proxy server. good thing i haven't banned any IPs yet, i would be banning my proxy server and locking everyone out including myself :O

**Comment by osusoy** [comment]
btw, i somehow got IP banned earlier with the expected effect. sorted it out on the DB but i think there is another bug hiding there.

**Comment by bluethrust** [comment]
Yeah, the IP I use is $_SERVER['REMOTE_ADDR'], just did a quick search and saw theres a $_SERVER['HTTP_X_FORWARDED_FOR'] variable that should solve the problem.  I am pretty sure most of the time I've used the $IP_ADDRESS variable that's set within the _setup.php file so I should be able to fix most of the problem by checking for the HTTP_FORWARDED otherwise use the REMOTE_ADDR one.  But there are a few scatters of just using the $_SERVER['REMOTE_ADDR'] so will be a couple of files to change.
